tests/dragonfly/acl_family_test.py

```diff
@@ -707,6 +710,10 @@ async def subscribe_worker(channel: aioredis.client.PubSub):
     subscriber_obj = subscriber.pubsub()
     await subscriber_obj.subscribe("channel")
 
+    # There's a rare timing issue if we don't wait here, but given the weak guarantees of Pub/Sub,
+    # that's probably OK.
+    await asyncio.sleep(1)
```

Comment:
If you look back on my PR that added the logs, I had the exact same suspicion. Although I added the `asyncio.sleep` on line 698 with hopes that the test would help the producer side (and then the subscriber would get all the messages). Little did I know that we also needed this here :)

Reply:
Yes, I agree that if it was a different command, it would have been a bug (that such a sleep is required). But Pub/Sub has very weak guarantees..